ssl/quic/quic_port.c

```diff
@@ -647,9 +658,164 @@ static int port_try_handle_stateless_reset(QUIC_PORT *port, const QUIC_URXE *e)
     return i > 0;
 }
 
-#define TOKEN_LEN (sizeof("openssltoken") + \
-                   QUIC_RETRY_INTEGRITY_TAG_LEN - 1 + \
-                   sizeof(unsigned char))
+static void cleanup_validation_token(QUIC_VALIDATION_TOKEN *token)
+{
+    OPENSSL_free(token->remote_addr);
+}
+
+/**
```

Comment:
rather question or note. Is there any guideline how/when doxy-gen like comments should be used?

In my opinion it makes sense to use doxy-gen comments to public functions or types (API), not sure how much useful it is to add those comments to functions which are private to modules.

Reply:
currently [coding style](https://openssl-library.org/policies/technical/coding-style/index.html) says we should consider adding them everywhere.  Ideally (speaking only for myself), I'd like to see them used universtally, so we can eventually build a doxygen html bundle to allow future developers to more easily navigate and understand our internals, in an effort to garner greater community participation from new individuals